### DISCUSSION TITLE
[FA23] October 18, 2023: Termination of Term Data Collection

### DISCUSSION THREAD

**Discussion by ewang2002** [UCSD-Historical-Enrollment-Data/UCSDHistEnrollData#13] [Data Updates]
UCSD ITS has blocked my VM's IP address and has asked me to discontinue the use of my scraper. Therefore, no further data will be collected for this term.

We anticipate running the scraper for WI24 as usual, but will need to make adjustments to the scraping process.

## Impact
All enrollment data has been collected (aside from some issues which have been addressed in previous announcements). Due to the early termination of the scraper, no further drop data (without/with W) will be collected.

**Comment by pink10000** [comment]
:(

**Comment by mr4tt** [comment]
):

**Comment by pink10000** [comment]
How long do you plan to maintain this project?

**Reply by ewang2002** [reply]
Hey @pink10000, sorry for the late reply.

This will be the last academic year I will be able to maintain this project. I might stop working on this project as early as the end of Winter 2024, but this is to be decided. 

I am currently looking for someone to take over this project once I am no longer able to maintain this project.
